### DISCUSSION TITLE
Shortcut to run program outside of sandbox?

### DISCUSSION THREAD

**Discussion by alpe12** [sandboxie-plus/Sandboxie#1618] [Documents and notes]
I have a folder with many things configured as ForceFolder and I would like to create an shortcut to an app inside it to run it outside sandbox.
Is it possible?
Something in "/box:" maybe?
Thanks.

**Comment by isaak654** [comment]
I suggest one of the following options:
* [StartCommandLine.html#disable-forced-programs](https://sandboxie-plus.github.io/sandboxie-docs/Content/StartCommandLine.html#disable-forced-programs)
* #1500 (see also [BreakoutProcess](https://sandboxie-plus.github.io/sandboxie-docs/Content/BreakoutProcess.html) and [BreakoutFolder](https://sandboxie-plus.github.io/sandboxie-docs/Content/BreakoutFolder.html) documentation)
